godtools/App/Share/SwiftUI Preferences/ViewGeometry.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewGeometry.swift
+//  godtools
+//
+//  Created by Levi Eggert on 6/12/22.
+//
+
+import SwiftUI
+
+struct ViewGeometry: View {
+    
+    var body: some View {
+        
+        GeometryReader { geometry in
+            Color.clear
+                .preference(key: ViewSizePreferenceKey.self, value: geometry.size)
```

Comment:
I'm a little unclear what this is for.  It just adds a clear background to fill the space basically?